libs/blocks/quiz/quizoption.js

```diff
@@ -3,25 +3,23 @@ import { html } from '../../deps/htm-preact.js';
 export const OptionCard = ({
   text, title, coverImage, cardIcon, options, disabled, selected,
 }) => {
-  const cardIconHtml = html`<div class="consonant-OneHalfCard-image">
-    <img src="${cardIcon}" alt="Video" class="icon" />
+  const cardIconHtml = html`<div class="quiz-option-icon">
+    <img src="${cardIcon}" alt="Quiz Option Icon" class="icon" />
```

Comment:
Same generic class question, `.icon` might have conflicts with other blocks

Reply:
no need for this class at all.